### PR TITLE
Kevin/hotfix ibpage signup button

### PR DESCRIPTION
**Pull request by kevinw-binary**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — COMMENTED** [review]
Hi @kevinw-binary , quick one, what is the task # in redmine?

**Comment by arome-binary** [comment]
> Hi @kevinw-binary , quick one, what is the task # in redmine?

Seen it, ignore the question.....

**Review by arome-binary — DISMISSED** [review]

**Review by ShuvoHabib — DISMISSED** [review]
LGTM

**Review by sean-binary — APPROVED** [review]

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -277,9 +277,10 @@ const SignupButton = styled(Button)`
     opacity: 0;
 `
 
-const LinkSignupButton = styled(LinkButton)`
```
Comment (sean-binary) [inline src/components/layout/nav.js:280]: It can't be extended from a button as it has the `to` and `external` prop behaving like a localizedlink.

**Review by sean-binary — CHANGES_REQUESTED** [review]
Left a comment.

**Review by sean-binary — APPROVED** [review]

File: src/components/localization/localized-link.js [hunk 2]
```diff
@@ -77,7 +77,7 @@ export const LocalizedLink = React.forwardRef(({ external, ...props }, ref) => {
     }, [])
 
     if (external || external === 'true') {
```
Comment (ShuvoHabib) [inline src/components/localization/localized-link.js:79]: Can we make it shorter? i.e: if (external) {}

File: src/components/layout/nav.js [hunk 3]
```diff
@@ -786,6 +787,7 @@ const StyledNavRight = styled(NavRight)`
     > a:last-child {
         pointer-events: ${(props) => (props.move ? 'visible' : 'none')};
         cursor: ${(props) => (props.move ? 'pointer' : 'default')};
+        opacity: ${(props) => (props.move ? 1 : 0)};
```
Comment (ShuvoHabib) [inline src/components/layout/nav.js:790]: If the reason of this opacity is to hide, perhaps we can use visibility: visible || hidden

**Review by ShuvoHabib — COMMENTED** [review]
Just a couple of minor feedback

**Review by nijil-binary — APPROVED** [review]

**Pull request merged**